internal/resource/erlang_cookie_test.go

```diff
@@ -67,7 +67,7 @@ var _ = Describe("ErlangCookie", func() {
 
 		It("creates an erlang cookie that is base64 encoded and 24 characters", func() {
 			cookie, ok := secret.Data[".erlang.cookie"]
-			Expect(ok).NotTo(BeFalse())
+			Expect(ok).To(BeTrue())
```

Comment:
OMG 🤣 😭

Reply:
Ginkgo linter is picky 🥲

Reply:
I was more laughing at the ridiculousness of the double negative in the old code